torch_glow/tests/nodes/quantize_dequantize_test.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch.nn.functional as F
+
+from tests.utils import jitVsGlow
+
+
+def test_quantization_basic():
+    """Basic test of the PyTorch quantize and dequantize Node on Glow."""
+
+    def test_f(a, b):
+        q = torch.nn.quantized.Quantize(255, 0, torch.quint8)
+        dq = torch.nn.quantized.DeQuantize()
+        return dq(q(a.add(b)))
```

Comment:
Why does this test add then do quant-dequant on empty network? Should it be dq(q(a)) or put the add in the middle of the quantized section?

Reply:
That would need a translation of quantized add operator as well, which introduced more non-stable stuff. I prefer to let the test of quantize-dequantize be like this, at least for now.